src/encoded/static/components/experiment.js

```diff
@@ -917,6 +927,19 @@ const ExperimentComponent = ({ context, auditIndicators, auditDetail }, reactCon
                 title={`Functional characterization experiments with ${context.accession} as an elements cloning`}
             />
 
+            {isFunctionalExperiment && plasmidMapDocs.length > 0 ?
+                <div>
```

Comment:
In general it's better to avoid empty wrapper `<div>`s. Normally you can use `<React.Fragment>` or in this case I'm not sure the `<div>`s are even necessary. However, it looks like this duplicates the pattern for biosample characterizations code (line 900) so probably this should be left as-is.